.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing
+
+The following is a set of guidelines for contributing to Hey-UI. Please spend several minutes reading these guidelines before you create an issue or pull request.
+
+## Code of Conduct
+
+We have adopted a [Code of Conduct](https://github.com/hey-car/hey-ui/blob/main/CODE_OF_CONDUCT.md) that we expect project participants to adhere to
```

Comment:
please change to: `we expect project participants to follow. Also, please read`...

Reply:
there's still a typo here, `elso`